src/cmap/connection.ts

```diff
@@ -892,15 +864,6 @@ export class ModernConnection extends TypedEventEmitter<ConnectionEvents> {
     this[kHello] = response;
   }
 
-  // Set the whether the message stream is for a monitoring connection.
-  set isMonitoringConnection(value: boolean) {
```

Comment:
Agreed, and also this connection implementation doesn't support streaming for monitoring yet. That is the next ticket. When it does we could consider reintroducting some skipping of hellos if we deem it necessary, but since we have the mode setting likely we won't.